src/Fmt.ml

```diff
@@ -238,14 +235,14 @@ let fill_text text =
       (String.split (String.rstrip text) ~on:'\n')
   in
   fmt_if (Char.is_whitespace text.[0]) " "
-  $ vbox 0
-      (hovbox 0
-         (list_pn lines (fun ?prev:_ curr ?next ->
-              fmt_line curr
-              $
-              match next with
-              | Some str when String.for_all str ~f:Char.is_whitespace ->
-                  close_box $ fmt "\n@," $ open_hovbox 0
-              | Some _ when not (String.is_empty curr) -> fmt "@ "
-              | _ -> noop)))
-  $ fmt_if (Char.is_whitespace text.[String.length text - 1]) " "
+  $ hovbox 0
+      ( list_pn lines (fun ?prev:_ curr ?next ->
+            fmt_line curr
+            $
+            match next with
+            | Some str when String.for_all str ~f:Char.is_whitespace ->
+                close_box $ fmt "\n@," $ open_hovbox 0
+            | Some _ when not (String.is_empty curr) -> fmt "@ "
+            | _ -> noop)
+      $ fmt_if (Char.is_whitespace text.[String.length text - 1]) " "
```

Comment:
Shouldn't this be `@ ` ?

Reply:
No because we don't want to break just before `*)`